tensorflow/core/kernels/mkl_conv_ops.cc

```diff
@@ -1576,65 +1576,66 @@ class MklQuantizedConv2DOp
     const float* max_filter = max_filter_vector.flat<float>().data();
 
     std::vector<mkldnn::primitive> net;
-    if (bias_enabled) {
-      if (std::is_same<Tbias, qint32>::value) {
-        return static_cast<Tbias*>(
-            const_cast<Tbias*>(bias_tensor.flat<Tbias>().data()));
-      }
+    if (!bias_enabled) {
```

Comment:
This and the next `if` can be moved to the beginning of the function (line 1566, before `int bias_index_offset`).

Reply:
@penpornk Modified by you request. Please check, thanks.